opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -162,20 +162,16 @@ message MetricDescriptor {
   }
   Type type = 4;
 
-  // The label keys associated with the metric descriptor. Keys in this list must
-  // be unique.
-  repeated string label_keys = 5;
+  // The set of constant label associated with the metric descriptor. Labels in this list
+  // apply to all data points.
+  repeated opentelemetry.proto.common.v1.StringKeyValue constant_labels = 5;
```

Comment:
Is listing labels here semantically equivalent to listing the same labels and values all data points and is merely a storage optimization or they are semantically different? I am asking because the comment here and the comment describing the meaning of `labels` field in the data point messages are not the same. This one does not mention say anything about whether this labels identify the timeseries or no.

Reply:
done.